src/czml3/properties.py

```diff
@@ -238,13 +241,34 @@ def __attrs_post_init__(self):
                 self.cartographicRadians,
                 self.cartesianVelocity,
                 self.reference,
+                self.interval,
             )
         ):
             raise ValueError(
-                "One of cartesian, cartographicDegrees, cartographicRadians or reference must be given"
+                "One of cartesian, cartographicDegrees, cartographicRadians, cartesianVelocity, epoch, interval or "
+                "reference must be  given "
             )
 
 
+@attr.s(str=False, frozen=True, kw_only=True)
+class Holes(BaseCZMLObject, Interpolatable, Deletable):
```

Comment:
There is no `Holes` property in https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/, here I see https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Polygon#properties it's a `PositionListOfLists`